ynput/ayon-frontend
# Browser: Filter products/versions list by taskType

## Description of changes

- Adds a filtering drop down above products/versions list on browser page that filters by taskType. 
- When no types are selected, nothing is filtered.
- Right click on a task(s) to filter by it.

## Testing

1. Go to `/projects/demo_Commercial/browser`
2. Select folder `/assets/characters/00_faatzieuck_moekseauld`
3. Select some options from **Task types...**
4. See versions in the middle filter down.
5. Right click on a task (bottom left) and click "Filter products by task" item.

## Notes

The toolbar is getting quite filled up now, maybe there will need to be a sub menu at some point.

## Additional context

https://github.com/ynput/ayon-frontend/assets/49156310/8e96405b-8793-4eff-b000-54816e4e2f25

![image](https://github.com/ynput/ayon-frontend/assets/49156310/72a4413a-487a-495e-9f4a-9909809a501d)
![image](https://github.com/ynput/ayon-frontend/assets/49156310/dbabe72a-ace0-4c8a-ae86-fe2b361b7bfc)
